federatedscope/core/auxiliaries/metric_builder.py

```diff
@@ -16,6 +16,6 @@ def get_metric(types):
     for func in register.metric_dict.values():
         res = func(types)
         if res is not None:
-            name, metric = res
```

Comment:
I just noticed that `get_metric` has no docstring and returns a dict that has such a special design of its values. Could you just supplement the missing docstring?